app/models/sample_manifest.rb

```diff
@@ -129,11 +129,32 @@ def default_filename
     "#{study_id}stdy_manifest_#{id}_#{created_at.to_formatted_s(:dmy)}"
   end
 
-  # Use a default value of 1 for rows_per_well if not set
+  # Number of rows per well in the manifest file, specified in manifest_types.yml.
```

Comment:
It's a good point - I have it on my to do list to ask what the number should be. It kind of has to be high enough, because the spreadsheets are locked down and won't let them insert new rows. This is for a good reason, because some data is created upfront on generation of the manifest (Sanger Sample Ids, Sample Manifest Assets), so if they could insert rows then this data wouldn't exist for them.